### ISSUE TITLE
[Bug] Possible accessibility issues found by Android Lint warnings; severity change could prevent future accessibility issues

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/fenix#15641]
## Steps to reproduce
- Run Android Lint: `./gradlew lintDebug`
- See "Accessibility" checks category on results page

### Expected behavior
No WARNINGS

### Actual behavior
Several WARNINGS:
> 1 | warning ClickableViewAccessibility: Accessibility in Custom Views
9 | warning ContentDescription: Image without contentDescription
4 | warning LabelFor: Missing accessibility label

In order, this could be indicative of:
- a view that cannot be accessed with accessibility tools turned on
- a view that will not speak when highlighted
- an editable field that doesn't speak clearly for accessibility issues

We should consider fixing these. Furthermore, if we change the severity of these checks (and other accessibility checks) to ERROR, **we can prevent these issues from occurring in the future before the PR merges.** Even if we don't fix these issues now, we could suppress them and change the severity so future issues are not introduced.

### Device information

* Android device: ?
* Fenix version: master-ish a173749ad8ae65a705ed091ea6931d09890fa330
